string printout_mode # Text, Binary
bool swap_beacons # true, false
uint8 ema_window # 0-255
uint16 beacons_span # 100-3000
